exporter/honeycombexporter/honeycomb.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2019 OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package honeycombexporter
+
+import (
+	"context"
+
+	"github.com/honeycombio/opentelemetry-exporter-go/honeycomb"
+	"github.com/open-telemetry/opentelemetry-collector/consumer/consumerdata"
+	"github.com/open-telemetry/opentelemetry-collector/exporter"
+	"github.com/open-telemetry/opentelemetry-collector/exporter/exporterhelper"
+	"github.com/open-telemetry/opentelemetry-collector/oterr"
+	"go.opentelemetry.io/otel/api/core"
+)
+
+const oTelCollectorUserAgentStr = "Honeycomb-OpenTelemetry-collector"
+
+type HoneycombExporter struct {
+	exporter *honeycomb.Exporter
+}
+
+func newHoneycombTraceExporter(cfg *Config) (exporter.TraceExporter, error) {
+	exporter, err := honeycomb.NewExporter(honeycomb.Config{
+		APIKey: cfg.APIKey,
+	},
+		honeycomb.TargetingDataset(cfg.Dataset),
+		honeycomb.WithAPIURL(cfg.APIURL),
+		honeycomb.WithUserAgentAddendum(oTelCollectorUserAgentStr),
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	hce := HoneycombExporter{
+		exporter: exporter,
+	}
+	return exporterhelper.NewTraceExporter(
+		cfg,
+		hce.pushTraceData,
+		exporterhelper.WithTracing(true),
+		exporterhelper.WithMetrics(false),
```

Comment:
this is an option to generate metrics for the exporter, it should be set to true. BTW, we are removing these options and settings the metrics to be "on" by default, but, no worries I will take care of that when it reaches the contrib repo

Reply:
> this is an option to generate metrics for the exporter, it should be set to true. BTW, we are removing these options and settings the metrics to be "on" by default, but, no worries I will take care of that when it reaches the contrib repo

I'll push a commit in a moment with the suggestions below, so might as well get this one too. Thanks!